pkg/model/apiversion.go

```diff
@@ -9,8 +9,7 @@ type APIVersion int
 
 const (
 	apiVersionUnknown APIVersion = iota // must be first
-	V1alpha1
-	V1beta1
+	V0
```

Comment:
@wdbaruni Are we okay calling this version of the spec `V0`?

Reply:
better to use `V1gamma`. V0 indicates it is the very first version but in this case it isn't